README.md

```diff
@@ -1,2 +1,61 @@
-# html-css-capstone
-A project for completion of HTML and CSS block at Microverse
+<h2>Psych Conference</h2>
+
+![Screenshot (45)](https://user-images.githubusercontent.com/69966775/124003051-63747900-d9ce-11eb-830e-f6d0c6c5144c.png)
+
+<h6>First Capstone Project</h6>
+
+Additional description about the project and its features.
+
+<h1>Built With</h1>
+
+- HTML
+- CSS
+- JS
+
+## Live demo
+
+This is the [live demo](https://sly-cloud.github.io/html-css-capstone/) of my capstone project.
+
+## Presentation Video
+
+This [video](https://www.loom.com/share/7117d7bbad67485dbe1d6c4bf97434a1) explains the aspects of the website and the idea behind it
+
+Getting Started
+
+<h2> To get a local copy of this file:</h2>
+
+1. Go to [repo](https://sly-cloud/html-css-capstone/).
+2. Click the **code** button (usually in green background)
+3. Open terminal on your local machine
+4. Enter `<git clone>`
+
+<h1>Authors</h1>
+
+👤 Adonike Sylvester
+
+- Github: [@Sly-Cloud](https://github.com/Sly-Cloud)
+- Twitter: [@SylvesterHills](https://twitter.com/SylvesterHills)
+- LinkedIn: [Sylvester Adonike](https://www.linkedin.com/in/sylvester-adonike/)
+
+🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://sly-cloud.github.io/html-css-capstone/issues).
```

Comment:
- [ ] This link seems to be broken. Kindly fix it.

Reply:
Done! 💯